lib/extensions/descendant_loader.rb

```diff
@@ -266,9 +277,7 @@ def clear
   end
 end
 
-ActiveRecord::Base.singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
-ActiveSupport::Dependencies.send(:prepend, DescendantLoader::AsDependenciesClearWithLoader)
-ActsAsArModel.singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
+DescendantLoader.setup
 
 at_exit do
```

Comment:
Technically, this should be part of the setup as well.

Reply:
Maybe address this comment?

Reply:
Agreed.   Will do.